actor_blockstore/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
I hope we can avoid a new crate here. The stores are for internal use only in testing, so can go in the same dependency we already use for that: runtime.

For now, I suggest just next to runtime/test_utils.rs. But we might also want to reorganise to create a subdirectory there for tests utils rather than that one giant file anyway.